pkgs/applications/emulators/dosbox-staging/default.nix

```diff
@@ -91,17 +91,17 @@ stdenv.mkDerivation rec {
     # original dosbox. Doing it this way allows us to work with frontends and
     # launchers that expect the binary to be named dosbox, but get out of the
     # way of vanilla dosbox if the user desires to install that as well.
-    mv $out/bin/dosbox $out/bin/${pname}
+    mv $out/bin/dosbox $out/bin/${self.pname}
     makeWrapper $out/bin/dosbox-staging $out/bin/dosbox
 
     # Create a symlink to dosbox manual instead of merely copying it
     pushd $out/share/man/man1/
-    mv dosbox.1.gz ${pname}.1.gz
-    ln -s ${pname}.1.gz dosbox.1.gz
+    mv dosbox.1.gz ${self.pname}.1.gz
+    ln -s ${self.pname}.1.gz dosbox.1.gz
```

Comment:
We could get away here with swapping the file and the symlink and reducing this to one command.

Reply:
Good idea!